drivers/audio/tlv320dac310x.c

```diff
@@ -33,8 +38,9 @@ struct codec_driver_data {
 
 static struct codec_driver_config codec_device_config = {
 	.i2c_device	= NULL,
-	.i2c_dev_name	= DT_CODEC_I2C_BUS_NAME,
```

Comment:
the change to use DT_CODEC_I2C_BUS_NAME instead of .i2c_dev_name/.i2c_address should NOT be part of this commit/PR.  Please remove them.

Its actually better to leave the code as is, so its closer to supporting multiple instances of this i2c device.

Reply:
Good catch. I have retained the I2C name/address in the structure and also added the GPIO pin/flags.